next.config.mjs

```diff
@@ -101,32 +98,20 @@ export default withSentryConfig(
     nextConfig,
   ),
   {
-    // For all available options, see:
-    // https://github.com/getsentry/sentry-webpack-plugin#options
-
-    // Suppresses source map uploading logs during build
     silent: true,
-    release: process.env.SENTRY_RELEASE,
-  },
-  {
-    // For all available options, see:
-    // https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/
-
-    // Upload a larger set of source maps for prettier stack traces (increases build time)
+    release: { create: false },
+    unstable_sentryWebpackPluginOptions: {
```

Comment:
removed [old env vars](https://github.com/r34son/profile/pull/422/files#diff-37f17b407bf98e4af95f60b4343b826e85184d6ff5ad2c55b64bfe28f701c423L11) for webpack define plugin in favor of new api